base/rootfs/etc/cont-init.d/00-banner.sh

```diff
@@ -3,6 +3,12 @@
 # Home Assistant Community Add-on: Base Images
 # Displays a simple add-on banner on startup
 # ==============================================================================
+
+# Exits early if not running as an add-on
+if bashio::var.is_empty "${SUPERVISOR_TOKEN:-}"; then
+    exit
+fi
```

Comment:
This image is designed around running as an add-on. I have no intention of supporting other use cases.